_post/2022-09-22-hyotester-reflection.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+author: tommytester
+title: "Tommy's Reflection on the Class So Far"
```

Comment:
Take a look at the author and title.  I believe they should be
~~~
author: hyotester
title: "Hyotester's Reflection on the class so far"
~~~
(or whichever title you choose using your name)